### PR TITLE
INT-1487 Add new query filter to crucial endpoints

### PR DESCRIPTION
**Pull request by nurutdinov**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -1983,6 +1983,10 @@ components:
       type: string
       description: A timezone value.
 
+    Filter:
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:1986]: can we maybe just add this parameter to `PageAndSort`? or make some other model like `FilteredPageAndSort` that will accumulate filter and `PageAndSort`?

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk-call-queue/talk-call-queue-api.yaml [hunk 2]
```diff
@@ -72,20 +72,10 @@ paths:
       description: Use this endpoint to retrieve a list of call queues. The operation supports paging and sorting.
```
Comment (olyavertwist) [inline talk-call-queue/talk-call-queue-api.yaml:72]:
```suggestion
      description: Use this endpoint to retrieve a list of call queues. The operation supports paging, sorting, and filtering.
```

**Review by olyavertwist — APPROVED** [review]
Just one optional change

File: talk-call-queue/talk-call-queue-api.yaml [hunk 3]
```diff
@@ -477,6 +467,12 @@ components:
       description: This parameter affects sorting. It specifies the property name to sort records, e.g., by price or date.
       schema:
         type: string
+    QueryFilter:
+      name: filter
+      description: This parameter controls filtering. Enter your query here, Inperium will look for matching records. For example, enter 'AND(eq(firstName,Bob),eq(lastName,Lee))' to filter out records with Bob Lee.
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk-call-queue/talk-call-queue-api.yaml:472]: Yes

File: talk-call-queue/talk-call-queue-api.yaml [hunk 4]
```diff
@@ -787,6 +783,23 @@ components:
           description: A text to be converted to a voice message or ID of an audio file.
           type: string
 
+    CallQueueParams:
+      type: object
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk-call-queue/talk-call-queue-api.yaml:787]: Sure, thanks!

**Pull request merged**